analysis/data/speed_rwanda.csv

```diff
@@ -0,0 +1,244 @@
+205,Kak City aka Kacyiru,,,['http://pbs.twimg.com/media/EUrKDzRUMAA2jjT.jpg']
+206,Kicukiro,"['Airtel', '3G']",4.5 Mbps,['http://pbs.twimg.com/media/EUrI9ddVAAQTEv0.jpg']
+207,,"['MTN', 'Rwanda😩']",1.7 Mbps (yarivuye kuri 490kbps),['http://pbs.twimg.com/media/EUrIwU-UUAIdwhy.jpg']
+208,,,,['http://pbs.twimg.com/media/EUrIfVxUYAIFXuY.jpg']
+209,Kimironko https://t.co/GhENnW2bxJ,['TrueConnect'],,['http://pbs.twimg.com/media/EUrH9pZUMAMcUJf.jpg']
+210,Kicukiro (Rebero),"['MTN', '3G']",N/A,['http://pbs.twimg.com/media/EUrIBacUYAAMBlm.jpg']
+211,Rebero,"['Liquid', 'Home']",,['http://pbs.twimg.com/media/EUrH7qrUcAAGUhN.jpg']
+212,Kicukiro ,"['Mtn', '3G']",3.2Mbps,['http://pbs.twimg.com/media/EUrHvKbUwAICMYC.jpg']
+213,Kicukiro,['Airtel'],1.3 Mbps,['http://pbs.twimg.com/media/EUrG2E-U8AAVe2t.jpg']
+214,Nyamirambo/ Nyakabanda     ,"['Mango', 'Telecom']",,['http://pbs.twimg.com/media/EUrGlodUYAAIdFl.jpg']
+215,Ruyenzi       ,['@airtelrw'],120 kbps⬇️ 850 kbps⬆️,['http://pbs.twimg.com/media/EUrFoS9UEAAsQsJ.jpg']
+216,Gisozi,['Airtel'],15Mbps,['http://pbs.twimg.com/media/EUrFg4mUwAAJIOl.jpg']
+217,Kimironko,"['Airtel', '3G']",1.6Mbps,['http://pbs.twimg.com/media/EUrFEJPU4AAo-YB.jpg']
+218,,,,['http://pbs.twimg.com/media/EUrETNfU0AAfHsR.png']
+219,Gikondo ,"['MTN', 'Rwanda', '(3G)']",3.8 Mbps,['http://pbs.twimg.com/media/EUrEJMcU8AAnAqE.jpg']
+220,Gatsata ,"['Mango', 'Telecom']",1.4 Mbps,['http://pbs.twimg.com/media/EUrDoO2UwAEShqY.png']
+221,,['MTN'],3.7 ,['http://pbs.twimg.com/media/EUrDpCHUwAEtUf6.jpg']
+222,Kagarama,[],65Kbps,['http://pbs.twimg.com/media/EUrDfprU4AAKiY7.jpg']
+223,Musanze,['Airtel'],,['http://pbs.twimg.com/media/EUrCbssVAAEAGEC.png']
+224,Kicukiro,['@airtelrw'],6.6 Mbps,['http://pbs.twimg.com/media/EUrCBrEUcAADFH_.jpg']
+225,Kimironko ,"['MTN', 'Rwanda']",17 Mbps 😀,['http://pbs.twimg.com/media/EUrBbiWU4AAzP2b.png']
+226,Kanombe- Nyarugunga https://t.co/LoQRcsdBs6,,,['http://pbs.twimg.com/media/EUrBwNOUwAAk0kG.jpg']
+227,Town,['MTN'],2.8 Mbps,['http://pbs.twimg.com/media/EUrBWPNUYAEIPVV.jpg']
+228,Kacyiru ,['Liquid'],,['http://pbs.twimg.com/media/EUrBTzAU0AEpaEM.jpg']
+229,Gikondo,[],3.9 Mbps (🙂),['http://pbs.twimg.com/media/EUrAO0iUMAAPuox.jpg']
+230,Gisozi | Gacinjiru       ,['@MTNRwanda'],170 Kbps ⬇️ 460 Kbps ⬆️ ,['http://pbs.twimg.com/media/EUrASm3U8AEFC1b.jpg']
+231,,[],330 Kbps (☹️),['http://pbs.twimg.com/media/EUq_kqvVAAEBxCq.png']
+232,Gikondo,['MTN'],3.9 Mbps,['http://pbs.twimg.com/media/EUq_sFgVAAIq24z.jpg']
+233,Rwamagana ,"['MTN', '3G']",3.5 Mbps,['http://pbs.twimg.com/media/EUq-MKtU8AAI532.png']
+234,,,,['http://pbs.twimg.com/media/EUq8kMyUUAA4dj4.jpg']
+235,,,,['http://pbs.twimg.com/media/EUq7L7SUcAA27tr.jpg']
+236,Nyakabanda,[],1.1 Mbps,
+237,,['Airtel'],9.9 Mbps,
+238,Mumena,"['Mango', 'Telecom']",8.5 Mbps,
+239,Gikondo,['MTN'],7.2 Mbps,
+240,Kimisagara,['Airtel'],1.2 Mbps,
+241,Gisozi,"['CB', 'Net']",34 Mbps,
+242,Kimironko,[],4.6 Mbps,
```

Comment:
@espoirMur It will be better just having the data as follow
full text (Location, speed, ISP) and images.  for example 
```
238,Mumena,"['Mango', 'Telecom']",8.5 Mbps,
239,Gikondo,['MTN'],7.2 Mbps,
240,Kimisagara,['Airtel'],1.2 Mbps,
241,Gisozi,"['CB', 'Net']",34 Mbps,
242,Kimironko,[],4.6 Mbps,
```
CB Net is one company and Mango telecom is also one company but the data here are being split. 
We can actually apply an algorithm in other to get the right telecommunication companies